packages/dd-trace/test/appsec/index.sequelize.plugin.spec.js

```diff
@@ -0,0 +1,93 @@
+'use strict'
+
+const path = require('path')
+const axios = require('axios')
+const getPort = require('get-port')
+const agent = require('../plugins/agent')
+const appsec = require('../../src/appsec')
+const Config = require('../../src/config')
+
+describe('sequelize', () => {
+  withVersions('sequelize', 'sequelize', sequelizeVersion => {
+    withVersions('mysql2', 'mysql2', () => {
+      withVersions('sequelize', 'express', (expressVersion) => {
+        let sequelize, User, server, port
+
+        // init tracer
+        before(async () => {
+          await agent.load(['express', 'http'], { client: false }, { flushInterval: 1 })
+          appsec.enable(new Config({
+            appsec: {
+              enabled: true,
+              rules: path.join(__dirname, 'express-rules.json'),
+              apiSecurity: {
+                enabled: true,
+                requestSampling: 1
+              }
+            }
+          }))
+        })
+
+        // close agent
+        after(() => {
+          appsec.disable()
+          return agent.close()
+        })
+
+        // init database
+        before(async () => {
+          const { Sequelize, DataTypes } = require(`../../../../versions/sequelize@${sequelizeVersion}`).get()
+
+          sequelize = new Sequelize('db', 'root', '', {
+            host: '127.0.0.1',
+            dialect: 'mysql'
+          })
+          User = sequelize.define('User', {
+            username: DataTypes.STRING,
+            birthday: DataTypes.DATE
+          })
+
+          await sequelize.sync({ force: true })
+          await User.create({
+            username: 'janedoe',
+            birthday: new Date(1980, 6, 20)
+          })
+        })
+
+        // clean database
+        after(async () => {
+          await User.drop()
+        })
+
+        // init express
+        before((done) => {
+          const express = require(`../../../../versions/express@${expressVersion}`).get()
+
+          const app = express()
+          app.get('/users', async (req, res) => {
+            const users = await User.findAll()
+            res.json(users)
+          })
+
+          getPort().then(newPort => {
+            port = newPort
+            server = app.listen(newPort, () => {
+              done()
+            })
+          })
+        })
+
+        // stop express
+        after(() => {
+          return server.close()
+        })
+
+        it('Should complete the request on time', (done) => {
+          axios.get(`http://localhost:${port}/users`)
+            .then(() => done())
```

Comment:
```suggestion
            .then(done)
```

Reply:
No, if `then` returns a value, it will call to `done(value)` which means that the test will fail with the error `value`. we need to call to `done()` without parameters if we want a success.